crates/test-cluster/src/lib.rs

```diff
@@ -960,6 +965,19 @@ impl TestClusterBuilder {
         self
     }
 
+    pub fn with_max_submit_position(mut self, max_submit_position: usize) -> Self {
+        self.max_submit_position = Some(max_submit_position);
+        self
+    }
```

Comment:
I used it initially, and then realized it wasn't needed after setting the latency step low. But it seems worth it to leave it in since ideally you'd be able to change anything via the builders.